service/initialise.go

```diff
@@ -39,11 +40,19 @@ var GetHealthCheck = func(cfg *config.Config, buildTime, gitCommit, version stri
 
 // GetZebedee gets the Zebedee Client
 var GetZebedee = func(cfg *config.Config) clients.ZebedeeClient {
+	if !cfg.EnableZebedeeCallbacks {
+		log.Info(context.Background(), "Zebedee callbacks are disabled, returning nil Zebedee client")
```

Comment:
should be lowercase, also consider putting the action first before the reason (same for dataset log below)
```suggestion
		log.Info(context.Background(), "returning nil zebedee client as callbacks are disabled")
```

https://github.com/ONSdigital/dp-standards/blob/main/LOGGING_STANDARDS.md#effective-event-types